src/cc2olx/olx.py

```diff
@@ -110,11 +114,37 @@ def _get_content(self, element_data):
         return content_type, details
 
     def _process_static_links(self, html):
+        """
+        Process static links like src and href to have appropriate links.
+        """
         srcs = re.findall(r'src\s*=\s*"(.+?)"', html)
-        for src in srcs:
-            if 'IMS-CC-FILEBASE' in src:
-                new_src = urllib.parse.unquote(src).replace("$IMS-CC-FILEBASE$", "/static")
-                html = html.replace(src, new_src)
+        hrefs = re.findall(r'href\s*=\s*"(.+?)"', html)
+
+        items = srcs + hrefs
+
+        def process_wiki_reference(item, html):
+            """
+            Replace $WIKI_REFERENCE$ with edx /jump_to_id/<url_name>
+            """
+            search_key = urllib.parse.unquote(item).replace("$WIKI_REFERENCE$/pages/", "")
+
+            # remove query params and add suffix .html to match with resource_id_by_href
+            search_key = search_key.split('?')[0] + '.html'
+            for key in self.cartridge.resource_id_by_href.keys():
+                if key.endswith(search_key):
+                    replace_with = '/jump_to_id/{}'.format(self.cartridge.resource_id_by_href[key])
+                    html = html.replace(item, replace_with)
+                    return html
+            logger.warn('Unable to process Wiki link - %s', item)
+            return html
+
+        for item in items:
+            if 'IMS-CC-FILEBASE' in item:
+                new_item = urllib.parse.unquote(item).replace("$IMS-CC-FILEBASE$", "/static")
+                html = html.replace(item, new_item)
```

Comment:
these two can also be moved in a function to keep things consistent

Reply:
Refactored